frontend/src/main.tsx

```diff
@@ -1,13 +1,15 @@
-import ReactDOM from 'react-dom/client';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'; // Correct imports
-import Layout from './Layout';
-import { Assignments, Grading, Settings, Dashboard } from './pages';
-import './index.css';
-import React from 'react';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+
+import { Assignments, Grading, Settings, Dashboard } from "./pages";
+import Layout from "./components/Layout";
+
+import "./global.css";
 
 export function App(): React.JSX.Element {
   return (
-    <Router basename="/khoury-classroom">
```

Comment:
Double check with Sebi on port and base-url rename-> Likely has implications for deployment